guide/ops/locations/_azure-ARM.md

```diff
@@ -94,3 +94,9 @@ The loginUser can be anything, as long as it's specified.
 
 The `overrideAuthenticateSudo: true` key tells Apache Brooklyn that default on Azure images do not have passwordless sudo 
 configured by default.
+
+#### Known issues
+There are currently two know issues with Azure ARM:
+
+* It can take a long time for VMs to be provisioned
+* Azure ARM appears to have some fairly strict rate limiting
```

Comment:
I would mention these are limits on the API usages rather than the number of machines and cause `AzureComputeRateLimitExceededException`s